text/0004-iso-8601.md

```diff
@@ -0,0 +1,51 @@
+- Start Date: 2023-10-06
+- Contributors:
+  - [Zak Burke](zburke@ebsco.com)
+- RFC PRs:
+  - DRAFT REVIEW: https://github.com/folio-org/rfcs/pull/13
+  - PRELIMINARY REVIEW: TBD
+  - PUBLIC REVIEW: TBD
+  - FINAL REVIEW: TBD
+- Outcome: <!-- Leave this blank. Will eventually be either ACCEPTED or REJECTED -->
+
+# Date-time values must comply with IETF RFC-3339
+
+## Summary
+
+* Validation of date-time values must assert compliance with IETF RFC-3339.
```

Comment:
The project [informally](https://issues.folio.org/browse/DMOD-256?focusedCommentId=17538&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-17538) made the RFC-3339 decision back in 2017